react/src/component/MovieComponent.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect} from 'react';
+import {useParams, useHistory} from 'react-router-dom';
+import { Image, Jumbotron} from 'react-bootstrap';
+import {getMovieById} from '../store/actions/MovieActions';
+import {useDispatch, useSelector} from 'react-redux';
+import {movieSelector} from '../store/selectors/MovieSelector';
+import Comment from './Comment';
+
+export default function MovieComponent() {
+
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    let { id } = useParams();
+
+    useEffect(() => {
+        dispatch(getMovieById(id));
+    },[]);
```

Comment:
```suggestion
    },[id]);
```

Mozda? Ako menjas po idu stranicu da nece poslati request